src/lib/__tests__/ThumbnailsSidebar-test.js

```diff
@@ -475,6 +477,14 @@ describe('ThumbnailsSidebar', () => {
             expect(stubs.removeClass).toBeCalledTimes(2);
             expect(stubs.addClass).toBeCalledTimes(1);
         });
+
+        test('should add the aria-current attribute when thumbnail is selected', () => {
+            thumbnailsSidebar.currentPage = 2;
+
+            thumbnailsSidebar.applyCurrentPageSelection();
+
+            expect(stubs.addAriaAttribute).toBeCalledTimes(1);
```

Comment:
Can we add an assertion for the `removeAttribute` call, as well, similar to the tests above?

Reply:
@jstoffan I just added another unit test case.